GVFS/GVFS.Common/FileSystem/PhysicalFileSystem.cs

```diff
@@ -177,6 +177,9 @@ public bool TryWriteTempFileAndRename(string destinationPath, string contents, o
             handledException = null;
             string tempFilePath = destinationPath + ".temp";
 
+            string parentPath = Path.GetDirectoryName(tempFilePath);

```

Comment:
@derrickstolee I ported your changes over here since I was spewing out errors.

Reply:
Good. Best to get the fix in sooner than #635.